client/go/outline/fetch.go

```diff
@@ -45,7 +45,7 @@ func fetchResource(url string) (string, error) {
 	resp.Body.Close()
 	if resp.StatusCode > 299 {
 		return "", platerrors.PlatformError{
-			Code:    platerrors.FetchConfigFailed,
+			Code:    platerrors.GenericErr,
```

Comment:
It might be useful to expose this error details? Especially the HTTP response code?